Update user guide to reflect current features in our application

kangzongxian opened the pull request:
This is to keep our user guide up to date.

kangzongxian committed:
Update user guide to reflect current features in our application.

bryanhce commented on docs/UserGuide.md:
```diff
@@ -22,17 +22,19 @@ guest management tasks done faster than traditional GUI apps.
 
-1. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
+1. Double-click the file to start the app. The GUI similar to the image below should appear in a few seconds. Note how the app contains some sample data.<br>
    ![Ui](images/Ui.png)
```
I was wondering if we should change the image? Because in the entry Bernice Yu the request is "hi" which does not make much sense. What do you think?

kangzongxian replied:
Agreed!

bryanhce commented on docs/UserGuide.md:
```diff
@@ -22,17 +22,19 @@ guest management tasks done faster than traditional GUI apps.
+   * **`add`**`n/John Doe p/98765432 e/johnd@example.com rm/05-73 dr/13/09/22 - 15/09/22 ng/3 rq/Extra towels`: Adds a guest named `John Doe` to the guest book.
 
-   * **`edit`** `1` `n/Bob rc/no`: Edits the 3rd guest shown in the current list.
+   * **`edit`** `1` `n/Bob rc/no rq/Room Service`: Edits the 3rd guest shown in the current list.
```
Should it be "Edits the first guest" instead of the third guest? Should we also spell it out since this is graded by the CS2101 side?

kangzongxian replied:
Agreed!

bryanhce commented on docs/UserGuide.md:
```diff
@@ -22,17 +22,19 @@ guest management tasks done faster than traditional GUI apps.
 
```
For the above, should we include the table of contents? Or should we wait till later to fill it up?

kangzongxian replied:
I think we can decide the content and fill it up after our meeting with everyone!

bryanhce reviewed (changes requested):
Some nits to fix!

kangzongxian committed:
Fix styling issues

bryanhce reviewed (approved):
LGTM

kangzongxian committed:
Fix merge conflicts

Merge branch 'master' into update-user-guide-quick-start

* master:
  Fix typos
  Fix small typo
  Rename dateRange to date
  Fix small spacing issues
  Update line breaks for MESSAGE_USAGE
  Update documentation
  Refactor into method
  Add find for other fields

# Conflicts:
#	docs/UserGuide.md
